cmd/util/ledger/util/registers/registers.go

```diff
@@ -184,24 +230,26 @@ func (a *AccountRegisters) Owner() string {
 }
 
 func (a *AccountRegisters) Payloads() []*ledger.Payload {
-	payloads := make([]*ledger.Payload, 0, len(a.registers))
-	return a.appendPayloads(payloads)
+	payloads := make([]*ledger.Payload, 0, a.Count())
+	a.insertPayloads(payloads)
+	return payloads
 }
 
-func (a *AccountRegisters) appendPayloads(payloads []*ledger.Payload) []*ledger.Payload {
+func (a *AccountRegisters) insertPayloads(payloads []*ledger.Payload) {
+	index := 0
```

Comment:
Maybe check if `len(payloads) == len(a.registers)` before `for` loop.

Reply:
Good idea! Added in 87ff7a2027fb45c84a1d7dc46fcce1084e05ba65